docs/features/circulation.md

```diff
@@ -0,0 +1,15 @@
+# Loan Circulation with Delivery Options and State Tracking
```

Comment:
```suggestion
# Circulation
```

I would explain a little bit more here, basically briefly listing all the circulation module features.
It feels like you are immediately jumping to the loan extension, while the core of the module is really the checkout/checking and the various states.